src/index.js

```diff
@@ -1,6 +1,78 @@
 /* eslint-env browser */
 
 import * as API from './interface.js'
+
+/**
+ * Gets cluster version
+ *
+ * @param {API.Config} cluster
+ * @param {API.RequestOptions} [options]
+ * @returns {Promise<string>}
+ */
+export const version = async (cluster, { signal } = {}) => {
+  const result = await request(cluster, 'version', {
+    method: 'GET',
+    signal
+  })
+
+  if (typeof result.version !== 'string') {
+    throw new Error(
+      `failed to parse version from response the body: ${JSON.stringify(
+        result,
+        null,
+        2
+      )}`
+    )
+  }
+
+  return result.version
+}
+
+/**
+ * Gets cluster information
+ *
+ * @param {API.Config} cluster
+ * @param {API.RequestOptions} [options]
+ * @returns {Promise<API.ClusterInfo>}
+ */
+export const info = async (cluster, { signal } = {}) => {
+  const result = await request(cluster, 'id', {
+    method: 'GET',
+    signal
+  })
+
+  const failure = result.error || result.ipfs?.error || ''
+  if (failure.length > 0) {
+    throw new Error(
+      `cluster id response has failure: ${JSON.stringify(result, null, 2)}`
+    )
+  }
+
+  const {
+    id,
+    addresses,
+    version,
+    commit,
+    peername,
```

Comment:
```suggestion
    peerName,
```

For consistency, it's mapped like this here already: https://github.com/nftstorage/ipfs-cluster/blob/824f8425247eb3707d8b9174058cd93dda79dfe5/src/index.js#L141

Reply:
```suggestion
    peername: peerName,
```